obs_status.yml

```diff
@@ -0,0 +1,1995 @@
+  mag_aca: 11.6
+  mag_aca_err: 0.2
+- agasc_id: 187829568
+  mag_aca: 11.2
+  mag_aca_err: 0.1
+- agasc_id: 188482256
+  mag_aca: 8.266
+  mag_aca_err: 0.1
+- agasc_id: 188501784
+  mag_aca: 10.453
+  mag_aca_err: 0.1
+- agasc_id: 188623712
+  mag_aca: 11.297
+  mag_aca_err: 0.1
+- agasc_id: 188750920
+  mag_aca: 11.469
+  mag_aca_err: 0.1
+- agasc_id: 195167088
+  mag_aca: 12.7
+  mag_aca_err: 0.1
+- agasc_id: 225576592
+  mag_aca: 11.297
+  mag_aca_err: 0.1
+- agasc_id: 239080480
+  mag_aca: 10.2
+  mag_aca_err: 0.1
+- agasc_id: 257953336
+  mag_aca: 11.594
+  mag_aca_err: 0.1
+- agasc_id: 264115840
+  mag_aca: 12.844
+  mag_aca_err: 0.1
+- agasc_id: 288753656
+  mag_aca: 11.359
+  mag_aca_err: 0.1
+- agasc_id: 306200176
+  mag_aca: 10.953
+  mag_aca_err: 0.1
+- agasc_id: 311824376
+  mag_aca: 11.25
+  mag_aca_err: 0.1
+- agasc_id: 327945264
+  mag_aca: 11.25
+  mag_aca_err: 0.1
+- agasc_id: 328077952
+  mag_aca: 11.016
+  mag_aca_err: 0.1
+- agasc_id: 330967424
+  mag_aca: 11.219
+  mag_aca_err: 0.1
+- agasc_id: 373564272
+  mag_aca: 11.578
+  mag_aca_err: 0.1
+- agasc_id: 377102496
+  mag_aca: 10.797
+  mag_aca_err: 0.1
+- agasc_id: 391120072
+  mag_aca: 11.297
+  mag_aca_err: 0.1
+- agasc_id: 396501520
+  mag_aca: 10.3
+  mag_aca_err: 0.1
+- agasc_id: 405271520
+  mag_aca: 11.2
+  mag_aca_err: 0.1
+- agasc_id: 405276920
+  mag_aca: 11.203
+  mag_aca_err: 0.1
+- agasc_id: 417611952
+  mag_aca: 5.0
+  mag_aca_err: 0.1
+- agasc_id: 450103168
+  mag_aca: 12.2
+  mag_aca_err: 0.1
+- agasc_id: 497824592
+  mag_aca: 13.078
+  mag_aca_err: 0.1
+- agasc_id: 505545160
+  mag_aca: 10.8
+  mag_aca_err: 0.1
+- agasc_id: 612370792
+  mag_aca: 12.797
+  mag_aca_err: 0.1
+- agasc_id: 614482704
+  mag_aca: 11.25
+  mag_aca_err: 0.1
+- agasc_id: 615257632
+  mag_aca: 11.031
+  mag_aca_err: 0.1
+- agasc_id: 615914552
+  mag_aca: 10.797
+  mag_aca_err: 0.1
+- agasc_id: 616699752
+  mag_aca: 11.484
+  mag_aca_err: 0.1
+- agasc_id: 626667256
+  mag_aca: 10.85
+  mag_aca_err: 0.1
+- agasc_id: 639635448
+  mag_aca: 11.2
+  mag_aca_err: 0.1
+- agasc_id: 643702120
+  mag_aca: 11.4
+  mag_aca_err: 0.1
+- agasc_id: 647762704
+  mag_aca: 11.453
+  mag_aca_err: 0.1
+- agasc_id: 649070072
+  mag_aca: 12.719
+  mag_aca_err: 0.1
+- agasc_id: 652480464
+  mag_aca: 10.075
+  mag_aca_err: 0.1
+- agasc_id: 653529952
+  mag_aca: 10.844
+  mag_aca_err: 0.1
+- agasc_id: 653919168
+  mag_aca: 13.1
+  mag_aca_err: 0.1
+- agasc_id: 654050016
+  mag_aca: 11.625
+  mag_aca_err: 0.1
+- agasc_id: 664279632
+  mag_aca: 11.672
+  mag_aca_err: 0.1
+- agasc_id: 686557584
+  mag_aca: 9.609
+  mag_aca_err: 0.1
+- agasc_id: 686557784
+  mag_aca: 9.047
+  mag_aca_err: 0.1
+- agasc_id: 689441328
+  mag_aca: 12.0
+  mag_aca_err: 0.1
+- agasc_id: 690625640
+  mag_aca: 11.2
+  mag_aca_err: 0.1
+- agasc_id: 693243160
+  mag_aca: 4.5
+  mag_aca_err: 0.1
+- agasc_id: 694561568
+  mag_aca: 9.8
+  mag_aca_err: 0.1
+- agasc_id: 720504096
+  mag_aca: 11.344
+  mag_aca_err: 0.1
+- agasc_id: 723651200
```

Comment:
For this one, I suppose I'm not expecting a mags entry on its own?  It looks to me like the star-obs data spits out 12.21 mag.